src/plugin/fieldViews/CustomFieldView.ts

```diff
@@ -12,7 +12,7 @@ export interface CustomFieldDescription<Data = unknown, Props = unknown>
   props: Props;
 }
 
-export const createCustomField = <Data, Props>(
+export const createCustomField = <Data, Props = unknown>(
```

Comment:
Why was this needed?

Reply:
Ah, yeah, good to document. At the moment, we must always pass a value for the `props` argument in `createCustomField – we cannot make it optional, or the type passed to the type parameter and the value passed to the function might disagree, giving us 

```
Type 'Props | undefined' is not assignable to type 'Props'.
  'Props' could be instantiated with an arbitrary type which could be unrelated to 'Props | undefined'.
```

It felt silly to write

```ts
createCustomField<Asset[], undefined>([], undefined)
```

So adding `unknown` here lets us write 

```ts
createCustomField<Asset[]>([], undefined)
```

But the ideal would be 

```ts
createCustomField<Asset[]>([])
```